### PR TITLE
- image upload done

### PR DESCRIPTION
**Pull request by swaponsust**
- image upload done
- New editor added
- those two value added when new post create

### CODE REVIEW

File: app/src/main/java/com/prominentdev/blog/activities/ActivityPost.java [hunk 1]
```diff
@@ -210,26 +247,51 @@ private void doMultiPartRequest(String file) {
...
+        RequestBody fbody = RequestBody.create(MediaType.parse("image"), partFile);
+
+
         Request request = new Request.Builder()
-                .addHeader("Content-Type", "application/octet-stream")
-                //.header("Content-Disposition", "file;filename=" + "img_0000001.jpg")
-                .addHeader("Google-Access-Token", sessionManager.getParticularField(SessionManager.ACCESS_TOKEN))
-                .addHeader("X-CSRF-Token", xCSRFToken)
+                .header("Content-Type", "application/octet-stream")
+                .header("Content-Disposition", "file; filename=\"newimg4.jpg\"")
```
Comment (ndubbaka) [inline app/src/main/java/com/prominentdev/blog/activities/ActivityPost.java]: @swaponsust We can't have all files to have same name newimg4.jpg.
Can we get file name from android for existing images in gallery. If image from camera doesn't have file name already can we automaticaly generate something like IMG_yyyyMMdd_kkmmssSSS.jpg whatever the standard is

File: app/src/main/java/com/prominentdev/blog/activities/ActivityPost.java [hunk 2]
```diff
@@ -279,12 +341,29 @@ private void makeBlogPost() {
...
+            JSONObject imageObject = new JSONObject();
+            imageObject.put("target_id", imageUploadResponse.getFid().get(0).getValue());
+            imageObject.put("description", "The most fascinating image ever!");
+            JSONArray imageArray = new JSONArray();
+            imageArray.put(imageObject);
+            jsonParams.put("field_image", imageArray);
+
+            //body input
+            JSONObject bodyObject = new JSONObject();
+            titleObject.put("value", knife.toHtml());
```
Comment (ndubbaka) [inline app/src/main/java/com/prominentdev/blog/activities/ActivityPost.java]: shouldn't this be bodyObject?

File: app/src/main/java/com/prominentdev/blog/activities/ActivityPost.java (hunk 1 above)
Comment (swaponsust) [reply app/src/main/java/com/prominentdev/blog/activities/ActivityPost.java]: @ndubbaka Yes sure. I'm changing it now.

File: app/src/main/java/com/prominentdev/blog/activities/ActivityPost.java (hunk 2 above)
Comment (ndubbaka) [reply app/src/main/java/com/prominentdev/blog/activities/ActivityPost.java]: @swaponsust shouldn't this be bodyObject?

Comment (swaponsust) [reply app/src/main/java/com/prominentdev/blog/activities/ActivityPost.java]: @ndubbaka let me check.

**Pull request merged**